patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -165,7 +165,7 @@
 +                items.add(itemstack);
              }
 +
-+            net.minecraftforge.event.ForgeEventFactory.fireBlockHarvesting(items, p_180657_1_, p_180657_3_, p_180657_1_.func_180495_p(p_180657_3_), 0, 1.0f, true, p_180657_2_);
++            net.minecraftforge.event.ForgeEventFactory.fireBlockHarvesting(items, p_180657_1_, p_180657_3_, p_180639_3_, 0, 1.0f, true, p_180657_2_);
 +            for (ItemStack stack : items)
```

Comment:
You have to edit the code in the Forge project that is generated by gradlew setupForge. Then gradlew genPatches will generate the patches.